### DISCUSSION TITLE
Utopia as a read-write storybook replacement

### DISCUSSION THREAD

**Discussion by wokalski** [concrete-utopia/utopia#1387] [Ideas]
Immediately after seeing utopia I thought it'd be neat to integrate it into our workflow. We're using storybook for our components but it is read-only. Something like utopia would lower the barrier of entry to modify the components and would tighten the feedback loop.

**Comment by maltenuhn** [comment]
This is a great idea, and actually a little bit like how we've been using Utopia to document our own components. Being able to see and play with the sample code is the single biggest thing we miss from Storybooks.
